### PR TITLE
chore: remove build args

### PR DESCRIPTION
**Pull request by dunglas**
Remove build args (as official images) and use the new installation method for `mlocati/php-extension-installer`.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,8 +20,7 @@ ENV APP_ENV=prod
 WORKDIR /srv/app
 
 # php extensions installer: https://github.com/mlocati/docker-php-extension-installer
-ADD https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
-RUN chmod +x /usr/local/bin/install-php-extensions
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
```
Comment (maxhelias) [inline Dockerfile:23]:
```suggestion
COPY --from=mlocati/php-extension-installer --link /usr/bin/install-php-extensions /usr/local/bin/
```

**Review by maxhelias — APPROVED** [review]
Thanks @dunglas

**Comment by thePanz** [comment]
So, supporting an "older" version of PHP would need more work?

Would you provide an example how to achieve it?

**Comment by maxhelias** [comment]
What do you mean ? You can always change the PHP version as before. Currently, how do you override the argument ?

**Comment by thePanz** [comment]
I use a Makefile with the following:

```
SERVER_NAME="blabla-bla.localhost"
PHP_VERSION="7.4"
PROJECT_NAME="blabla.bla"
CLI_PARAMS="--project-name=$(PROJECT_NAME)"

.PHONY: build
build:
    SERVER_NAME="${SERVER_NAME}" docker-compose $(CLI_PARAMS) build --build-arg PHP_VERSION=${PHP_VERSION}

**Comment by maxhelias** [comment]
Indeed, your use case is a good one for rollback for me. What do you think @dunglas ?

**Pull request merged**